cmd/experimental/kjobctl/config/samples/volume_bundle.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: kjobctl.x-k8s.io/v1alpha1
```

Comment:
How is this file loaded? Does it require a manual non-documened step? If documented, please point me where is the command which applies this file.
Also, can we can name it "ray_volume_bundle.yaml` to reflect this is only for the Ray mode?

Reply:
> How is this file loaded?

It should be included on volumeBundles in ApplicationProfile. I added it on `application_profile.yaml`. Thanks.

Reply:
> Also, can we can name it "ray_volume_bundle.yaml` to reflect this is only for the Ray mode?

Renamed.

Reply:
Thanks!